pkg/azclient/Makefile

```diff
@@ -73,21 +73,23 @@ generate: install-dependencies build generatecode generateimpl vet-all
 .PHONY: generatecode
 generatecode: build ## Generate client
 	$(TYPESCAFFOLD) --package github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice/v2 --package-alias armcontainerservice --resource ManagedCluster --client-name ManagedClustersClient --apiversion 2021-10-01
-	$(TYPESCAFFOLD) --package github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v2 --package-alias network --resource PrivateLinkService --client-name PrivateLinkServicesClient --verbs get,createorupdate,delete,list --apiversion 2021-08-01 --expand
 	$(TYPESCAFFOLD) --package github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources --package-alias resources --resource Deployment --client-name DeploymentsClient --verbs delete --apiversion 2017-05-10
-	$(TYPESCAFFOLD) --package github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v4 --package-alias armcompute --resource Disk --client-name DisksClient --apiversion 2022-03-02
+	$(TYPESCAFFOLD) --package github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v4 --package-alias armcompute --resource Disk --client-name DisksClient --apiversion 2022-07-02

```

Comment:
are we able to reuse the same API Version from the SDK directly?

Reply:
yes. I'll raise another pr